src/ansible_navigator/actions/exec.py

```diff
@@ -16,23 +17,43 @@
 
 GeneratedCommand = Tuple[str, Optional[List[str]]]
 
+logger = logging.getLogger(__name__)
 
-def _generate_command(exec_command: str, exec_shell: bool) -> GeneratedCommand:
+
+def _generate_command(
+    exec_command: str,
+    exec_shell: bool,
+    extra_args: Union[Constants, List[str]],
+) -> GeneratedCommand:
     """Generate the command and args.
 
     :param exec_command: The command to run
     :param exec_shell: Should the command be wrapped in a shell
+    :param extra_args: Any unknown or extra arguments passed on the command line
     :returns: The command and any pass through arguments
     """
-    pass_through_args = None
+    logger.debug("exec_command: %s", exec_command)
+    logger.debug("exec_shell: %s", exec_shell)
+    logger.debug("extra_args: %s", extra_args)
     if exec_shell and exec_command:
         command = "/bin/bash"
-        pass_through_args = ["-c", exec_command]
+        # Determine if any extra args were picked up
+        if isinstance(extra_args, list):
+            pass_command = exec_command + " " + " ".join(extra_args)
```

Comment:
Would this be better
```suggestion
            pass_command = " ".join((exec_command, *extra_args))
```
?

Reply:
way better.............. I didn't like it when I typed it.  ty

Reply:
There's another, a little better, suggestion. Did you miss it?

Reply:
Ah, I see you applied it already :)